packages/driver/tsconfig.json

```diff
@@ -2,7 +2,8 @@
   "extends": "@repo/tsconfig/base.json",
   "compilerOptions": {
     "outDir": "./dist",
-    "downlevelIteration": true
+    "downlevelIteration": true,
+    "noUnusedLocals": false
```

Comment:
Can we just remove the unused local (or add a tsignore) instead of setting this to false?

Reply:
hm there are quite a lot of unused vars in codecs/registry and I can't find a way to supres TS checks for the whole file just for this rule, but all these unused vars are also checked with eslint if they are not prefixed with _

Reply:
Yeah, it _is_ annoying, but I agree with @jaclarke that we should keep this usage. I'd rather us use `ts-expect-error`, fwiw. In the future, we'll actually use these fields and we can remove these rule exceptions.

Reply:
@scotttrinh ah okay, I think u said few months ago to try to remove all `ts-expect-error` lines, and use _ for all unused things (maybe it was for another project). I'll revert for this use case. 

Also, I may am a bit confused what we wanna have as a TS error and what as a lint error.